helm/hpcc/templates/thor.yaml

```diff
@@ -221,6 +225,7 @@ data:
     metadata:
       name: thorworker-%jobname
     spec:
+      {{- include "hpcc.placementMatchName" (dict "root" $ "pod" "thorworker-" "group" "thor") | indent 6 }}
```

Comment:
again, if want all have 2 thor's, "hungrythor" and "tinythor", and want different placements, then need to be able to differentiate this "pod" name, so needs .name as part of it as with other std. components.

Reply:
I add <cluster name> prefix for each Job: <cluster name>-<job prefix>-